args.py

```diff
@@ -20,33 +61,37 @@ def parse_args():
                         help='Print all validation predicitons to console')
 
     # main folder for data storage
-    parser.add_argument('--root-dir', type=str, default=None)
+    parser.add_argument('--root-dir', type=str, default='ecg_data')
 
     # model settings
-    parser.add_argument('--arch', type=str,
-                        help='type of architecture to be used, e.g. cnn1d')
     parser.add_argument('--model-name', type=str,
                         help='type of model to be used. Particular instance of a given architecture, e.g. cnn1d_3')
+    parser.add_argument('--version-latest', type=bool, default=False,
+                        help='Use the latest version of the model if available')
+    parser.add_argument('--version-bump', type=bool, default=False,
+                        help='Create a new version of the model')
+    parser.add_argument('--version', type=int, default=None,
+                        help='Explicitly choose the version')
+    
```

Comment:
Why don't you make this one parameter instead of 3? Look at the --resume arg, it uses both strings and values

Reply:
Done